src/index.html

```diff
@@ -15,15 +15,12 @@
   <meta property="og:image:height" content="630">
   <meta property="fb:app_id" content="397128703965407">
   <link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fullPage.js/2.8.9/jquery.fullPage.min.css">
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css">
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap3-dialog/1.34.7/css/bootstrap-dialog.min.css">
   <link rel="stylesheet" href="style.css">
 </head>
 <body>
 
-  <div id="website-content" class="hidden">
-
```

Comment:
Restore this because it was fixing the flickering on load

Reply:
Done in b827b8e.